packages/util-user-agent-browser/src/index.ts

```diff
@@ -15,5 +15,9 @@ export function appendToUserAgent(
   request: HttpRequest,
   userAgentPartial: string
 ): void {
-  request.headers["X-Amz-User-Agent"] += ` ${userAgentPartial}`;
+  if(request.headers["X-Amz-User-Agent"]) {
+    request.headers["X-Amz-User-Agent"] += ` ${userAgentPartial}`;
+  } else {
+    request.headers["X-Amz-User-Agent"] = `${userAgentPartial}`;
```

Comment:
Quotes are not required for assignment in else block
```suggestion
    request.headers["X-Amz-User-Agent"] = userAgentPartial;
```